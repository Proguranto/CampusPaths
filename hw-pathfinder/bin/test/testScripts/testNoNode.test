# Find a path with Dijkstra's algorithm when
# the start or dest node is not in the graph.

CreateGraph graph8

AddNode graph8 S
AddNode graph8 1
AddNode graph8 2
AddNode graph8 D

AddEdge graph8 S 1 1.0
AddEdge graph8 S 2 1.0
AddEdge graph8 1 D 1.0
AddEdge graph8 2 D 1.0

FindPath graph8 3 D

FindPath graph8 S 3

FindPath graph8 3 4
