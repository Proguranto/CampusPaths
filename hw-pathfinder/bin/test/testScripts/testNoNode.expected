# Find a path with Dijkstra's algorithm when
# the start or dest node is not in the graph.

created graph graph8

added node S to graph8
added node 1 to graph8
added node 2 to graph8
added node D to graph8

added edge 1.000 from S to 1 in graph8
added edge 1.000 from S to 2 in graph8
added edge 1.000 from 1 to D in graph8
added edge 1.000 from 2 to D in graph8

unknown: 3

unknown: 3

unknown: 3
unknown: 4
